lib/rubocop/cop/style/yoda_condition.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for Yoda conditions where the two parts of an expression
+      # are reversed from the typical order in a conditional statement.
```

Comment:
What makes one order more typical than another?  Consider re-wording this to explain that the goal is to enforce that the order match the way that humans (or maybe just English-speaker?) speak.